src/Router.ts

```diff
@@ -18,6 +30,17 @@ const deleteMovieController = new DeleteMovieController(movieService);
 const getMoMovieByIdController = new GetMovieByIdController(movieService);
 
 const router = Router();
+router.use("/movies", authenticateJWT);
```

Comment:
poderia separar os metodos http em um modulo separado pra centralizar dai tu pode so chamar aqui 

```typescript
// movies.ts

const express = require('express');
const router = express.Router();

router.post("/", (request: Request, response: Response) => {
  return createMovieController.handle(request, response);


module.exports = router;

```

```suggestion
router.use("/movies", authenticateJWT, moviesModule);
```